cards/cards.css

```diff
@@ -0,0 +1,236 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Outlined");
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Round");
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+:root{
+    --secoundary:#b3b3b4;
+    --secoundary-black:#272727;
+    --primary:#ff3e6c;
+    --gray-text:#535665;
+    --success:green;
+    --warning:#f9a61a;
+    --danger:#f34648;
+    --success-light:rgb(156, 216, 156);
+    --warning-light:#f8e0b7;
+    --danger-light:#f7b2b3;
+}
+body{
+    margin:1rem;
+    font-family: 'Roboto', sans-serif;
+}
+.component-heading{
+    border-bottom: 2px solid var(--primary);
+    width:fit-content;
+}
+.component-description{
+    color:var(--gray-text);
+    margin:1rem 0;
+}
+.highlight{
+    color:var(--primary);
+}
+.carbon-code{
+    box-sizing: border-box;
+    width: 100%;
+    min-height: fit-content;
+    max-height:20rem;
```

Comment:
On desktop site your carbon code looks too small 
![Untitled](https://user-images.githubusercontent.com/89142612/151790985-db27d974-33e1-4084-84b4-1d1683b0dd2c.png)
maybe you can change the height here